src/pp_27_tic_tac_toe_draw.py

```diff
@@ -0,0 +1,67 @@
+from pp_26_check_tic_tac_toe import get_winner
+
+game = [[0, 0, 0],
+	    [0, 0, 0],
+	    [0, 0, 0]]
+
+print(game[0])
+print(game[1])
+print(game[2])
+
+PLAYER_1 = 'x'
+PLAYER_2 = 'o'
+
+def player_move(player_input: str) -> dict:
+    """input tranformation into indices
+
+    Args:
+        player_input (str): user input
+
+    Returns:
+        dict: row index, column index
+    """
+    player_split = player_input.split(",", 1)
+    player_row = int(player_split[0]) - 1
+    player_col = int(player_split[1]) - 1
+    return {
+        "row" : player_row,
+        "column" : player_col
+        }
+
+
+def play_the_game(board: list):
+    """game board filling
+
+    Args:
+        board (list): input board
+    """
+    number_of_moves = 0
+    current_player = PLAYER_1
+    while True:
+        if number_of_moves >= 9:
+            break
+
+        print("Make your move as row number,column number")
+        player_input = input(f"Player {current_player}, you are on - row/column: ")
```

Comment:
I think it looks good. 👍

What do you think about adding a validation for user input? It should prevent user to enter place, which is already taken or choose place out of the board.